### PR TITLE
easyovs python3 support

### PR DESCRIPTION
**Pull request by haojue**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,8 +41,8 @@
     license='BSD',
     install_requires=[
         'oslo.config>=1.0,<=1.15',
-        'python-keystoneclient>=1.0,<=1.6',
```
Comment (yeasy) [inline setup.py:44]: why remove those dependencies?

File: Makefile [hunk 2]
```diff
@@ -28,7 +28,7 @@ codecheck: $(PYSRC)
 errcheck: $(PYSRC)
 	-echo "Running check for errors only"
 	pyflakes $(PYSRC)
-	pylint -E --rcfile=.pylint $(PYSRC)
```
Comment (yeasy) [inline Makefile]: Why disable lint?

File: easyovs/cli.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def __init__(self, stdin=sys.stdin, foreground=True):
             debug("username = %s\n" % os.getenv('OS_USERNAME') or
                   cfg.CONF.OS.username)
             passwd = os.getenv('OS_PASSWORD') or cfg.CONF.OS.password
-            passwd = passwd[:len(passwd)/4] + "****" + passwd[-len(passwd)/4:]
```
Comment (yeasy) [inline easyovs/cli.py]: Why comment out this line?

File: setup.py (hunk 1 above)
Comment (haojue) [reply setup.py:44]: sorry, did not check it when I created the PR, that was changed several months ago:)
for openstack Ussuri, it is using python3-keystoneclient-4.0.0, is it ok to remove just the upper constraint like <=1.6?

File: Makefile (hunk 2 above)
Comment (haojue) [reply Makefile]: > Why disable lint?

changed it back not to disable lint. I had vague memory when I installed it several months ago I may have some issue related to it, for simplicity I just disabled this lint back then.

File: easyovs/cli.py (hunk 3 above)
Comment (haojue) [reply easyovs/cli.py]: > Why comment out this line?

changed it back, just my makeshift way when installed it several months ago:)

**Review by yeasy — APPROVED** [review]

**Pull request merged**